lib/paper_trail/version_concern.rb

```diff
@@ -385,7 +385,7 @@ def sibling_versions
     #
     # @api private
     def version_limit
-      klass = item.class
```

Comment:
Calling `item` here is essentially what was causing the N+1 query, since instantiating the version's `item` requires querying the database for the `item`'s attributes.

This change avoids that database query by instead just referencing the `item_subtype` or `item_type` fields of the version record, which does not require any additional database query.